src/Sulu/Component/HttpCache/HttpCache.php

```diff
@@ -93,6 +93,6 @@ private function getReverseProxyTtl(Response $response)
             return $maxAge - $response->getAge();
         }
 
-        return null;
+        return;
```

Comment:
However in this case it should indeed be `null` as this method does return a value.